docs/program-setup/mbed_targets.md

```diff
@@ -281,9 +281,9 @@ The `detect_code` contains four ASCII characters containing only hexadecimal val
 
 The `OUTPUT_EXT` property controls the file type emitted for a target by the build system. You may set `OUTPUT_EXT` to `bin` for binary format, `hex` for Intel HEX format and `elf` for ELF format. We discourage using the `elf` value for `OUTPUT_EXT` because the build system must always emit an ELF file.
 
-### default_lib
+### c_lib
 
-The `delault_lib` property controls which library, small or standard, the `GCC_ARM` toolchain links. The `default_lib` property may take on the values `std` for the standard library and `small` for the reduced size library.
+The `c_lib` property controls which library, small or standard, the `GCC_ARM` toolchain links. The `c_lib` property may take on the values `std` for the standard library and `small` for the reduced size library.
```

Comment:
This is no longer specific to GCC_ARM only.

Reply:
I'll wait for this to be resolved before merging